.github/workflows/build-release.yml

```diff
@@ -9,32 +9,55 @@ env:
 
 jobs:
   build_and_publish:
-    runs-on: ubuntu-latest
     environment: release
+    runs-on: ubuntu-latest
+    name: "Release: build, sign and upload the app"
+    strategy:
+      matrix:
+        php-versions: ['7.4']
+        nextcloud: ['v21.0.0beta6']
+        database: ['sqlite']
+        experimental: [false]
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-versions }}
+
+      - name: Set up server non MySQL
+        uses: SMillerDev/nextcloud-actions/setup-nextcloud@main
         with:
-          path: ${{ env.APP_NAME }}
-      - name: Run build
-        run: cd ${{ env.APP_NAME }} && make && make appstore
+          cron: true
+          version: ${{ matrix.nextcloud }}
+          database-type: ${{ matrix.database }}
+
+      - name: Copy APP to the apps/ directory
```

Comment:
This is already done by the checkout+server action AFAIK.

Reply:
I will try

Reply:
Yea it did work, pushed a new commit